lib/cn/csched_sp3.c

```diff
@@ -445,7 +446,9 @@ sp3_log_progress(struct cn_compaction_work *w, struct cn_merge_stats *ms, bool f
         ms->ms_vblk_read2_wait.op_cnt, ms->ms_vblk_read2_wait.op_time,
         ms->ms_kblk_read.op_cnt, ms->ms_kblk_read.op_size, ms->ms_kblk_read.op_time,
         ms->ms_kblk_read_wait.op_cnt, ms->ms_kblk_read_wait.op_time,
-        ms->ms_vblk_wasted_reads, qt, pt, bt, ct);
+        ms->ms_vblk_wasted_reads, qt, pt, bt, ct
+        // clang-format on
+        );
```

Comment:
After closing paren line, for consistency with other use-cases...

Reply:
fixed and pushed.